site/specs/webrtc.yml

```diff
@@ -2421,6 +2532,198 @@ components:
           example: participant1
         deviceApiVersion:
           $ref: '#/components/schemas/deviceApiVersionEnum'
+    createDialogRequest:
+      type: object
+      description: The payload for a request to create a Dialog
+      properties:
+        dialogType:
```

Comment:
We don't need the dialog prefix. Just type will work.

Calling it state data is a little weird, especially for a creation request. I understand why you made a separate subtype, but honestly we could just have one type that includes the dialog type, the to, from, and user to user

Perhaps we can discuss this